### PR TITLE
GRASS GIS 7.8.5 release news

### PR DESCRIPTION
**Pull request by neteler**
- a new screenshot would be welcome
- please review and improve as needed

### CODE REVIEW

**Comment by veroandreo** [comment]
The release announcement seems fine to me :) What about using this screenshot: https://grass.osgeo.org/learn/gallery/#lidar-support&gid=1&pid=97 ??

File: content/news/2020_12_21_GRASS_GIS_7_8_5_released.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+---
+
+### GRASS GIS 7.8.5 released
+
+------------------------------------------------------------------------
+**What's new in a nutshell**
+
+As a follow-up to the previous GRASS GIS 7.8.4 we have published the
+<a href="/images/news/vnetdistance.png">
+  <img src="/images/news/vnetdistance.png"
```
Comment (neteler) [inline content/news/2020_12_21_GRASS_GIS_7_8_5_released.md:15]:
```suggestion
<a href="/images/gallerylidar/zmi_stone_town_opendronemap_v_surf_icw.png">
  <img src="/images/gallerylidar/zmi_stone_town_opendronemap_v_surf_icw.png"
```

File: content/news/2020_12_21_GRASS_GIS_7_8_5_released.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### GRASS GIS 7.8.5 released
+
+------------------------------------------------------------------------
+**What's new in a nutshell**
+
+As a follow-up to the previous GRASS GIS 7.8.4 we have published the
+<a href="/images/gallerylidar/zmi_stone_town_opendronemap_v_surf_icw.png">
+  <img src="/images/gallerylidar/zmi_stone_town_opendronemap_v_surf_icw.png"
+   alt="Vector network analysis in GRASS GIS" title="Vector network analysis in GRASS GIS"
```
Comment (neteler) [inline content/news/2020_12_21_GRASS_GIS_7_8_5_released.md:16]:
```suggestion
   alt="Zanzibar Mapping Initiative data processed in OpenDroneMap and interpolated respecting building footprints using v.surf.icw" title="Zanzibar Mapping Initiative data processed in OpenDroneMap and interpolated respecting building footprints using v.surf.icw"
```

File: content/download/linux.en.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ---
 title: "Linux"
```
Comment (neteler) [inline content/download/linux.en.md:2]:
```suggestion
title: "GRASS GIS for Linux"
```

File: content/download/linux.en.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
-date: 2020-10-10T11:02:05+02:00
+date: 2020-12-21T11:02:05+02:00
 description: "Download GRASS GIS packages for your favorite distribution"
```
Comment (neteler) [inline content/download/linux.en.md:4]:
```suggestion
description: "Download GRASS GIS packages for your favorite Linux distribution"
```

File: content/download/mac.en.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
 ---
 title: "Mac"
```
Comment (neteler) [inline content/download/mac.en.md:2]:
```suggestion
title: "GRASS GIS for Mac"
```

File: content/download/windows.en.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Windows"
```
Comment (neteler) [inline content/download/windows.en.md:2]:
```suggestion
title: "GRASS GIS for Windows"
```

**Comment by landam** [comment]
@neteler WinGRASS packages (standalone + osgeo4w) uploaded.

**Comment by neteler** [comment]
Fedora 32 + 33 + 34 + EPEL8 in testing (https://src.fedoraproject.org/rpms/grass) - takes 7 days, but we do not have to wait.

So I'll merge :-)

**Pull request merged**